third_party/move/move-prover/boogie-backend/src/bytecode_translator.rs

```diff
@@ -1505,6 +1783,31 @@ impl<'env> FunctionTranslator<'env> {
                         };
                         emitln!(writer, "{} := {}->{};", dest_str, src_str, field_sel);
                     },
+                    GetVariantField(mid, sid, variants, inst, field_offset) => {
+                        let inst = &self.inst_slice(inst);
```

Comment:
Same here.  It seems odd that `GetField` doesn't need to instantiate while you need to do it here.